certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -8958,7 +10068,8 @@
     },
     "DtoFillPdfForm": {
       "required": [
-        "FileName"
+        "FileName",
+        "formData"
```

Comment:
@Encodian The input field has been changed from optional to required. This might break existing flows. can you please confirm that this will not affect existing users.

Reply:
Setting the 'formData' property to required is not an issue because if the value is not provided the action would always fail with a 500 error. All this change does is assure a 400 Bad Request error is generated as opposed to a 500 Internal Server Error.